### PR TITLE
Update framework

### PR DESCRIPTION
**Pull request by peppy**
Prereqs:
- [x] https://github.com/ppy/osu-framework/pull/5397

### CODE REVIEW

**Comment by peppy** [comment]
Hmm, so with the new handling framework-side, everything seems to be pivoting on the `IsPresent` check to keep things working.

Previously, the logic in `DrawableStoryboardSprite`:

https://github.com/ppy/osu/blob/824e68dab35f114c1960c5a5c30dacbbe3c99896/osu.Game/Storyboards/Drawables/DrawableStoryboardSprite.cs#L70-L71

would result in a `ScreenSpaceDrawQuad` with zero width, but now it is non-zero.

The `IsPresent` check does save the day here, but I'm still not sure how correct this is as a solution.

**Comment by smoogipoo** [comment]
> would result in a `ScreenSpaceDrawQuad` with zero width, but now it is non-zero.

Can you explain how? Based on that code I'm not seeing it, because I'd think that _previously_ it would be non-zero and _now_ it's zero.

**Comment by peppy** [comment]
I haven't investigated, but it is 100% repro on that test if you want to take a look (before [6aac0bd](https://github.com/ppy/osu/pull/20179/commits/6aac0bd4e96e9da903c22a86162b596108847335))

File: osu.Game.Tests/Visual/Gameplay/TestSceneDrawableStoryboardSprite.cs [hunk 1]
```diff
@@ -78,8 +78,9 @@ public void TestZeroScale()
 
             AddStep("allow skin lookup", () => storyboard.UseSkinSprites = true);
             AddStep("create sprites", () => SetContents(_ => createSprite(lookup_name, Anchor.TopLeft, Vector2.Zero)));
+            AddAssert("sprites present", () => sprites.All(s => s.IsPresent));
             AddStep("scale sprite", () => sprites.ForEach(s => s.VectorScale = new Vector2(0, 1)));
-            AddAssert("zero width", () => sprites.All(s => s.ScreenSpaceDrawQuad.Width == 0));
+            AddAssert("sprites not present", () => sprites.All(s => s.IsPresent));
```
Comment (smoogipoo) [inline osu.Game.Tests/Visual/Gameplay/TestSceneDrawableStoryboardSprite.cs:83]: The assert description doesn't match the logic.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Review by smoogipoo — DISMISSED** [review]

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**